src/lib/replication/service/raft_repl_service.cpp

```diff
@@ -93,9 +93,12 @@ void RaftReplService::start() {
                         .with_hb_interval(HS_DYNAMIC_CONFIG(consensus.heartbeat_period_ms))
                         .with_max_append_size(HS_DYNAMIC_CONFIG(consensus.max_append_batch_size))
                         .with_log_sync_batch_size(HS_DYNAMIC_CONFIG(consensus.log_sync_batch_size))
+    // TODO fix the log_gap thresholds when adding new member.
+#if 0
```

Comment:
New member stops syncing from leader due to these thresholds. This needs to be debugged and fixed in a later commit.

Reply:
If we dont truncate, raft only does incremental resync. This is the log of the new member added to the group.

```
[09/19/24 12:05:40-07:00] [I] [test_raft_repl_dev_dynamic_replica_3] [297795] [raft_repl_dev.cpp:79] [rd=rdev1] Started New RaftReplDev group_id=3b25627e-f07b-41e8-b992-4696be833582, replica_id=00030123-4567-89ab-cdef-01234567
89ab, raft_server_id=558356 commited_lsn=0, compact_lsn=0, checkpoint_lsn:0, next_dsn=0 log_dev=1 log_store=0
....
[09/19/24 12:05:40-07:00] [D] [test_raft_repl_dev_dynamic_replica_3] [297760] [raft_repl_dev.cpp:1133] [rd=rdev1] Raft channel: Received 64 append entries on follower from leader, term 0, lsn 1 ~ 64 , my commited lsn 0 , leade
r commmited lsn 1004
```

Reply:
Can we put the comment why it is disabled, also we need to create an issue to track this, put in the description what is now happening after disable this, and what was happening without this.

Reply:
Ack. Have an issue created https://github.com/eBay/HomeStore/issues/561

Reply:
No. Looks like there is some error in nuraft/nuraft_mesg after sending the first pack. Created a issue to debug this with nuraft. Logs are attached to that issue.